components/org.wso2.transport.file/src/main/java/org/wso2/transport/file/connector/sender/VFSClientConnector.java

```diff
@@ -232,6 +233,40 @@ public boolean send(CarbonMessage carbonMessage, CarbonCallback carbonCallback,
                                     }
                                 }
                             }
+                        } else if (Constants.MODE_TYPE_BINARY_CHUNKED.equalsIgnoreCase(mode)) {
+                            inputStream = path.getContent().getInputStream();
+                            // Read a file in BINARY_CHUNKED mode
+                            long size = path.getContent().getSize();
+                            int bufferSize = Integer.parseInt(map.get(Constants.BUFFER_SIZE_IN_BINARY_CHUNKED));
+                            logger.debug("Reading " + path.getName() + " on " + mode + ". Chunk size: " + bufferSize +
+                                    " bytes, File size: " + size + " bytes.");
+                            byte[] buffer = new byte[bufferSize];
+                            int sequenceNumber = 1;
+                            int n1;
```

Comment:
@Ajanthy shall we rename this variable ?

Reply:
fixed in ad48c42b6aa3b2f3fd11165ea5bfa6039ed6bc72